src/App.tsx

```diff
@@ -1,61 +1,136 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number) {
+  const foundedUser = usersFromServer.find(user => user.id === userId);
+
+  return foundedUser || null;
+}
+
+const todosWithUser = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [option, setOption] = useState('0');
+  const [titleInputValue, setTitleInputValue] = useState('');
+  const [userIsSelected, setUserIsSelected] = useState(true);
+  const [titleOnPaper, setTitleOnPaper] = useState(true);
+  const [allTodos, setAllTodos] = useState(todosWithUser);
+
+  const addTodo = () => {
+    const maxId = Math.max(...allTodos.map(todo => todo.id));
+
+    return ({
+      id: maxId + 1,
+      title: titleInputValue,
+      completed: false,
+      userId: +option,
+      user: getUser(+option),
+    });
+  };
+
+  const handleChangeTitleInput = (event: { target: { value: string }; }) => {
+    const { value } = event.target;
+
+    setTitleInputValue(value);
+    setTitleOnPaper(true);
+  };
+
+  const handleChangeUserSelect = (event: { target: { value: string }; }) => {
```

Comment:
```suggestion
  const handleChangeUserSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
```
Add correct types everywhere